asm-utils/Cargo.toml

```diff
@@ -1,7 +1,11 @@
 [package]
 name = "powdr-asm-utils"
-version = "0.1.0"
-edition = "2021"
+description = "powdr asm utilities"
```

Comment:
These are utilities for riscv- (and in the future valida / llvm) assembly. Not sure if we should clarify

Reply:
I'll add some more words